src/shell.ts

```diff
@@ -20,13 +20,13 @@ import { ChildProcess } from 'child_process';
 import * as shelljs from 'shelljs';
 
 const WINDOWS = 'win32';
-const MACOS  = 'darwin';
-const LINUX  = 'linux';
+const MACOS = 'darwin';
+const LINUX = 'linux';
 
 export enum Platform {
-    WINDOWS,
-    MACOS,
-    LINUX
+    WINDOWS = 1,
+    MACOS = 2,
+    LINUX = 3
```

Comment:
based on https://www.tutorialsteacher.com/typescript/typescript-enum , we can set only the first numeric enum value.
so would be:
```suggestion
    WINDOWS = 1,
    MACOS,
    LINUX
```